src/dataviews/dataview-model-base.js

```diff
@@ -225,6 +263,7 @@ module.exports = Model.extend({
     return this.getSourceId() === this.layer.get('source');
   },
 
+  // @return might return undefined if there's no source
```

Comment:
Right, this comment was written before I added the guard in the initialize too. I'll 🔥